mlflow/server/auth/basic_auth.ini

```diff
@@ -1,3 +1,5 @@
 [mlflow]
 default_permission = READ
 database_uri = sqlite:///basic_auth.db
+root_username = root
```

Comment:
we need a way to create, or let user create, the first admin. Built in admin user is common, e.g., in [Argo CD](https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/)

Reply:
The built-in admin sounds good!